### ISSUE TITLE
General usage issues

### ISSUE THREAD

**Issue by tcrosen** [FacultyCreative/ngActiveResource#54]
I would love to use this library but I'm have a lot of issues working with it.  One of the biggest problems is the inconsistency between the README and the example code.  

For starters, the only dependency mentioned is `lodash`, yet it also seems to require `active-support` and `async` to run, and also downloads `meld` (though it doesn't appear to require this).  

The README also shows a completely different syntax for declaring models from the example code.  I'm finding I can get one way to work at runtime but fails at testing and the other works for testing but fails at runtime.  

There are a lot of desirable features in this library but right now it's a bit messy to use and that's holding me back from actively working with it.  If you have some time please give the docs and examples some love, thanks!

**Comment by evanhobbs** [comment]
I agree -- this is sort of a bummer because it looks like it could be pretty awesome but in it's present form it's basically unusable... I'm giving up in frustration after spending several hours trying to get it to work.

**Comment by PatrickJS** [comment]
+1

**Comment by AlJohri** [comment]
+1

**Comment by aarondufall** [comment]
+1

**Comment by AlJohri** [comment]
https://github.com/jmdobry/angular-data
https://github.com/mgonto/restangular

**Comment by adambarthelson** [comment]
Guess you can't really beat a good old fashioned $http

**Comment by 9mm** [comment]
Yes. very frustrating getting it to work, I'm not sure if there are any alternatives that offer associations?

**Comment by AlJohri** [comment]
http://angular-data-next.pseudobry.com/documentation/guide/angular-data-resource/relations

**Comment by lucalenardi** [comment]
+1

**Comment by kyleget** [comment]
+1

**Comment by leonpapazianis** [comment]
Hi guys, is it possible to add custom validators? I am looking at the documentation and cant really find something.
